app/src/main/java/org/wikipedia/editactionfeed/provider/MissingDescriptionProvider.kt

```diff
@@ -62,8 +62,8 @@ object MissingDescriptionProvider {
                     ServiceFactory.get(WikiSite(Service.WIKIDATA_URL))
                             .getWikidataLabelsAndDescriptions(StringUtils.join(qNumbers, '|'))
                 }
-                .map<Pair<String, PageTitle>> { response ->
-                    var sourceDescriptionAndTargetTitle: Pair<String, PageTitle>? = null
+                .map<Pair<PageTitle, PageTitle>> { response ->
+                    var sourceDescriptionAndTargetTitle: Pair<PageTitle, PageTitle>? = null
```

Comment:
It would be better if also changes the name from `sourceDescriptionAndTargetTitle` to `sourceAndTargetPageTitles`

Reply:
NVM, I've made a follow-up commit f72fc98 in PR #142